examples/async-examples.ts

```diff
@@ -0,0 +1,53 @@
+import {RevAiApiClient, RevAiJobOptions} from 'revai-node-sdk';
+
+/* Initialize your client with your revai access token */
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiApiClient(token);
+
+/* Get account details */
+var account = await client.getAccount();
+console.log(`Account: ${account.email}`);
+console.log(`Balance: ${account.balance_seconds} seconds`);
+
+/* Submit job and receive job object
+ * Media may be submitted from a local file or from a url by using:
+ * client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+ */
+var jobOptions = new RevAiJobOptions(
```

Comment:
Where are the job options used.

Reply:
This options implentation is incorrect, and it has been addressed. They are to be used in `submitJobLocalFile`